### PR TITLE
Refactor ssh key resource and documentation

### PR DESCRIPTION
**Pull request by Diptipowervs**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

**Comment by Diptipowervs** [comment]
--- PASS: TestAccIBMPIKey_basic (26.13s)
PASS

File: website/docs/r/resource_key.html.markdown [hunk 1]
```diff
@@ -31,7 +30,8 @@ resource "ibm_resource_key" "resourceKey" {
 }
 ```
```
Comment (ismirlia) [inline website/docs/r/resource_key.html.markdown:32]: This is not our file, our documentation files always start pi_*

**Comment by Diptipowervs** [comment]
--- PASS: TestAccIBMPIKey_basic (32.71s)
PASS

**Review by michaelkad — APPROVED** [review]
lgtm

**Comment by ismirlia** [comment]
Opened: https://github.com/IBM-Cloud/terraform-provider-ibm/pull/5239 closing this one

**Pull request closed without merging**